<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="1" failures="0" name="ConfigTest.ConfigReadTest-20150929143239" tests="1" time="0.002">
	<properties/>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
	<testcase classname="ConfigTest.ConfigReadTest" name="testConfig" time="0.002">
		<error message="global name 'data' is not defined" type="NameError">
<![CDATA[Traceback (most recent call last):
  File "/home/saikappa/workspace/SciPass/python/t/ConfigTest.py", line 22, in setUp
    config = str(os.getcwd()) + "/t/etc/Simple_SciPass_balancer_only.xml" )
  File "./SciPass.py", line 51, in __init__
    self._processConfig(self.configFile)
  File "./SciPass.py", line 516, in _processConfig
    json.dump(data, config)
NameError: global name 'data' is not defined
]]>		</error>
	</testcase>
</testsuite>
